src/main/webapp/WEB-INF/view/login.jsp

```diff
@@ -30,6 +30,7 @@
       <a href="/login">Login</a>
     <% } %>
     <a href="/about.jsp">About</a>
+    <a href="/admin">Administration</a>
```

Comment:
should this also be blocked off by an if-statement? Also this may just be a change you are planning to do incrementally so sorry if I'm jumping the gun.

Reply:
@CaelumTerrae  yes!!! thanks so much for noticing this, I will make that change!